api/cookies/cookie-clearer/popup.js

```diff
@@ -0,0 +1,100 @@
+const form = document.getElementById("control-row");
+const go = document.getElementById("go");
+const input = document.getElementById("input");
+const message = document.getElementById("message");
+
+// The async IIFE is necessary because Chrome <89 does not support top level await.
+(async function initPopupWindow() {
+  let [tab] = await chrome.tabs.query({ active: true, currentWindow: true });
+
+  if (tab && tab.url) {
```

Comment:
Not required, but you could also do `if (tab?.url) {` now.

Reply:
"Optional Chaining reached Stage 3 at the July 2019 TC39 meeting." [Chrome Status](https://www.chromestatus.com/feature/5748330720133120)

I'll update to include this.